SConstruct

```diff
@@ -325,7 +324,7 @@ env.Execute(Chmod(w_prefix_dir+'/bin/nexus-config', 0o755))
 nexus = env.Program('bin/nexus', ['source/nexus.cc']+src)
 
 TSTDIR = ['utils',
-	  'example']
+	      'example']
```

Comment:
The alignment has been changed.

Reply:
I'm not sure how to fix it, because in my editor it appears as aligned.

Reply:
I fixed it using another editor.